accepted/2023/net8.0-polyfills/net8.0-polyfills.md

```diff
@@ -0,0 +1,29 @@
+# .NET 8.0 Polyfill
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst)
+
+This document covers which APIs we intend to ship for older versions of .NET,
+which includes .NET Standard and .NET Framework.
+
+## Polyfills
+
+| Polyfill     | Assembly                   | Package                    | Existing? | API                    | Contacts                   |
+| ------------ | -------------------------- | -------------------------- | --------- | ---------------------- | -------------------------- |
+| TimeProvider | Microsoft.Bcl.TimeProvider | Microsoft.Bcl.TimeProvider | No        | [dotnet/runtime#36617] | [@tarekgh] [@geeknoid]     |
```

Comment:
Can we avoid perpetuating the BCL acronym? I don't think an existing package is a good enough reason. BCL is only familiar to .NET aficionados and a descriptive term like libraries, core libraries, platform, core, etc would be fine. Note we chose to use SYSLIBnnnn for codes not BCLnnnn

Reply:
We have shipped number of poly-fill packages with this naming pattern: `Microsoft.Bcl.AsyncInterfaces`, `Microsoft.Bcl.HashCode`, `Microsoft.Bcl.Compression`, `Microsoft.Bcl.Cryptography`, `Microsoft.Bcl.Json.Sources`, `Microsoft.Bcl.Async`.

Reply:
IIRC, the naming pattern is hideous on purpose.